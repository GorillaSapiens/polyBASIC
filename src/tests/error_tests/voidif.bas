if void == 5 goto 100
100 print
