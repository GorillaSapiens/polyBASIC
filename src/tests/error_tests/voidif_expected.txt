ERROR: @1:4, LEFT / RIGHT OPERATION MISMATCH ❰VOID❱ ❰INTEGER❱
